tests/unit/models_containers_test.py

```diff
@@ -95,6 +95,7 @@ def test_create_container_args(self):
             ulimits=[{"Name": "nofile", "Soft": 1024, "Hard": 2048}],
             user='bob',
             userns_mode='host',
+            uts='host',
```

Comment:
Can we have an integration test as well? They're a much better indicator of things working as intended for this kind of thing.

Reply:
I have included a simple integration test, `test_create_with_uts_mode`, similar in style to the others container API tests already present.